src/runtime/components/Form.vue

```diff
@@ -231,6 +232,7 @@ async function onSubmitWrapper(payload: Event) {
     emits('error', errorEvent)
   } finally {
     loading.value = false
+    setTimeout(() => activeElement?.focus(), 0)
```

Comment:
I'm not confortable with this setTimeout, I think we might be better off introducing a props like the button component `loadingAuto` to allow users to disable the automatic loading / disabled state.